server/src/internalClusterTest/java/org/opensearch/remotestore/RestoreShallowSnapshotV2IT.java

```diff
@@ -773,24 +818,169 @@ public void testInvalidRestoreRequestScenarios() throws Exception {
         indexDocuments(client, index, numDocsInIndex, numDocsInIndex + randomIntBetween(2, 5));
         ensureGreen(index);
 
-        // try index restore with remote store disabled
+        // try index restore with USER_UNREMOVABLE_SETTINGS setting disabled
```

Comment:
nit: We're trying to ignore the setting rather than disable anything
```suggestion
        // try index restore with USER_UNREMOVABLE_SETTINGS setting ignored
```